value.go

```diff
@@ -640,6 +640,11 @@ func (vlog *valueLog) Close() error {
 			err = terr
 		}
 	}
+	if vlog.discardStats != nil {
+		if terr := vlog.discardStats.Close(-1); terr != nil && err == nil {
```

Comment:
what does -1 mean?

Reply:
-1 means that we don't want to truncate the file. Just Sync and unmap it. 
https://github.com/dgraph-io/ristretto/blob/8ec1dc18f880d5e12cda4f6da65141f049dc4e02/z/file.go#L185-L203

Reply:
Got it.